### PR TITLE
Add Aug 2021 Cabal Notes

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Add the meeting notes from the August 2021 Podman Community
Cabal Meeting.

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

### CODE REVIEW

File: community/meeting/notes/2021-08-19/index.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Podman 4.0-dev is now upstream.
+Paul Holzinger has added a large change for Networks.
+More performance analysis and attempting to lessen memory and CPU usage.  Adopting Buildkit functionality in Buildah and thus Podman build.
+
+Giuseppe is working with supporting virtual pools to retrieve just files that are not already present in local storage, to help decrease load times.  It may not be Docker compatible, it may have to be OCI based only.
+
+We're looking for ideas/changes that might require breaking API changes.  But are hoping not to make too many at once.
+
+#### Podman on Widows (12:30 in the video)
```
Comment (lsm5) [inline community/meeting/notes/2021-08-19/index.md]: Windows, unless there's a new OS :)

File: community/meeting/notes/2021-08-19/index.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+Erik asked if systemd is working?  (Not likely to at the moment.)  He too uses Podman on Windows and it works fine for him now.
+
+WSL2 is installed on windows by default already in the latest, and then install Fedora from Microsoft store, and then Podman ran from there.
+
+Docker has a GUI interface that can be used from Windows, we would probably not provide a similar out of the box.
+
+If you create a container currently in Windows using the Fedora, you can't talk to the container outside of that Windows host.  Something that will need looking at.
+
+Fedora costs $10 for Fedora 34 distribution from the Microsoft Store.
```
Comment (lsm5) [inline community/meeting/notes/2021-08-19/index.md:48]: woah, is that like a 1-time fee?

File: community/meeting/notes/2021-08-19/index.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+Two upgrade paths in Windows per Pavel.  We'd like to know how the upgrade could happen seamlessly for the end-user.
+
+Docker checks the version at starti-up and then asks the user to do update.  Information is stored in a small json file.  They apparently do an update in a separate VM.
+
+On Docker, can you do a volume mount on a Windows directory?  Giuillaume says it does work.
+
+#### Open discussion (39:45 in the video)
+
+When's Podman v3.3 coming out?  Hopefully Monday, Aug 23, 2021.  Then we will likely be creating a Podman 3.4 for sometime later in the fall.
```
Comment (lsm5) [inline community/meeting/notes/2021-08-19/index.md:62]: Do we need to update this with current info?

File: community/meeting/notes/2021-08-19/index.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+One thing to watch is that Podman v4.0 can not break Fedora 35.  Fedora 36 should be in April 2022 and would be the target if we break Fedora 35, but that's hopefully won't be the case.
```
Comment (lsm5) [inline community/meeting/notes/2021-08-19/index.md]: but that* hopefully

**Review by lsm5 — CHANGES_REQUESTED** [review]
some comments, but mostly LGTM.

File: community/meeting/notes/2021-08-19/index.md (hunk 3 above)
Comment (TomSweeneyRedHat) [reply community/meeting/notes/2021-08-19/index.md:62]: No, I think it represented our knowledge at the time.

File: community/meeting/notes/2021-08-19/index.md (hunk 2 above)
Comment (TomSweeneyRedHat) [reply community/meeting/notes/2021-08-19/index.md:48]: Actually, $9.99 + tax, or $10.61 in my case, and yes, one time cost.  I'm not sure if it's one time per machine or per account....

File: community/meeting/notes/2021-08-19/index.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply community/meeting/notes/2021-08-19/index.md]: tweaked.

File: community/meeting/notes/2021-08-19/index.md (hunk 4 above)
Comment (TomSweeneyRedHat) [reply community/meeting/notes/2021-08-19/index.md]: fixed

**Pull request merged**